sidebar.css

```diff
@@ -0,0 +1,84 @@
+.docs-container {
+  max-width: 1280px;
+  width: 100%;
+  margin: 0 auto;
+}
+
+.docs-content {
+  margin-top: 2rem;
+}
+
+.docs-sidebar {
+  flex: 0 0 200px;
+  margin-right: 5rem;
+  background-color: var(--color-white);
+}
+
+.list-topic {
+  padding: 1rem;
+  font-weight: var(--font-medium);
+  border-bottom: 1px solid var(--color-white);
+  margin-bottom: 0.5rem;
+}
+
+.list-topic a {
+  text-decoration: none;
+  color: var(--color-black);
+  font-weight: var(--font-bold);
+  font-size: 1rem;
+}
+
+.list-subtopic {
+  padding: 0.4rem 0.8rem;
+  margin-bottom: 0.5rem;
+  transition: all 0.3s ease-in;
+  font-size: 1rem;
+}
+
+.list-subtopic .active {
+  font-weight: var(--font-bold);
+  color: var(--default-color);
+}
+
+.list-subtopic li {
+  font-size: 1rem;
+  list-style: none;
+  padding: 0.7rem 1.1rem;
+  font-weight: var(--font-medium);
+  transition: all 0.3s ease-in-out;
+}
+
+.list-subtopic li a:hover {
+  cursor: pointer;
+  color: var(--color-default);
+}
+
+.list-subtopic a {
+  display: flex;
+  align-items: center;
+  color: var(--color-shadow);
+  text-decoration: none;
+  transition: all 0.3s ease-in-out;
+}
+
+@media screen and (max-width: 800px) {
+  nav {
```

Comment:
Use a class named as sidebar-nav and pass the properties mentioned in it and use this in your html file.

Reply:
Actually, this isn't sidebar nav the only nav I've used is the one that holds the logo and other links on top.